app/src/main/java/de/danoeh/antennapod/fragment/preferences/MainPreferencesFragment.java

```diff
@@ -107,10 +108,16 @@ private void setupMainScreen() {
             IntentUtils.openInBrowser(getContext(), "https://forum.antennapod.org/");
             return true;
         });
+        findPreference(PREF_CONTRIBUTE).setOnPreferenceClickListener(preference -> {
+            IntentUtils.openInBrowser(getContext(), "https://antennapod.org/contribute/");
+            return true;
+        });
         findPreference(PREF_SEND_BUG_REPORT).setOnPreferenceClickListener(preference -> {
             startActivity(new Intent(getActivity(), BugReportActivity.class));
             return true;
         });
+
+
```

Comment:
Could you please remove the empty lines again?

Reply:
Sure i'll do that soon.
About the icon. Don't worry, i already considered that issue before choosing the current icon. The wrench doesn't necessarily mean coding. It could mean anything that helps 'build' the project. Trust me, most aren't going to think it means coding, espcially non coders. When paired with the text, its meaning is obvious. I say leave it.